dev/io.openliberty.checkpoint_fat_springboot_data/publish/servers/checkpointSpringBootData/server.xml

```diff
@@ -0,0 +1,25 @@
+<!-- 
+ (C) Copyright IBM Corporation 2023.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<server>
+	<applicationManager startTimeout="90s"/>
+	<featureManager>
+		<feature>springBoot-3.0</feature>
```

Comment:
I guess it wouldn't matter much but the war test would not require the springBoot-3.0 feature.

Reply:
It does not harm and allows me to keep a single server.xml config for the test.